File: src/main/java/de/thatsich/intellie/common/registries/GuiRegistry.java
package de.thatsich.intellie.common.registries;

import cpw.mods.fml.common.network.IGuiHandler;
import cpw.mods.fml.common.network.NetworkRegistry;
import de.thatsich.intellie.common.ABaseMod;
import de.thatsich.intellie.common.util.logging.ILog;
import net.minecraft.entity.player.EntityPlayer;
import net.minecraft.world.World;

import javax.inject.Inject;
import javax.inject.Singleton;

@Singleton
public class GuiRegistry implements IGuiHandler
{
	private final ILog log;

	@Inject
	GuiRegistry( final ILog log )
	{
		this.log = log;
	}

	public void init( ABaseMod instance )
	{
		NetworkRegistry.INSTANCE.registerGuiHandler( instance, this );
	}

	@Override
	public Object getServerGuiElement( int ID, EntityPlayer player, World world, int x, int y, int z )
	{
		return null;
	}

	@Override
	public Object getClientGuiElement( int ID, EntityPlayer player, World world, int x, int y, int z )
	{
		return null;
	}
}

File: src/main/scala/de/thatsich/minecraft/intellie/common/registries/GuiRegistry.scala
package de.thatsich.minecraft.intellie.common.registries

import cpw.mods.fml.common.network.{NetworkRegistry, IGuiHandler}
import net.minecraft.world.World
import net.minecraft.entity.player.EntityPlayer
import de.thatsich.minecraft.intellie.common.ABaseMod

/**
 *
 *
 * @author thatsIch
 * @since 07.04.2014.
 */
class GuiRegistry
	extends IGuiHandler
{
	def init(instance: ABaseMod)
	{
		NetworkRegistry.INSTANCE.registerGuiHandler(instance, this)
	}

	def getServerGuiElement(ID: Int, player: EntityPlayer, world: World, x: Int, y: Int, z: Int): Object =
	{
		null
	}

	def getClientGuiElement(ID: Int, player: EntityPlayer, world: World, x: Int, y: Int, z: Int): Object =
	{
		null
	}
}
